rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -530,8 +513,55 @@ private static int checkMethod(Map<String, String> paths,
         return 1;
     }
 
+/*
+    TODO delete when the TCK is updated
+
+ This code (which replaces getStatus below) has two issues:
+  - it incurs an unneccessary performance hit due to the extra network call.
+  - it does not guarantee that an LRA will be valid when the JAX-RS method executes since it can be
+ closed at any time during the method invocation. The only way to guarantee validity of an LRA during a method invocation is
+ to enlist with it and any attempt to join with an inactive LRA will fail with 412 and that is good enough.
+
+ Therefore remove the following invalid TCK tests:
+    TckLRATypeTests.mandatoryWithClosedLRA
+    TckLRATypeTests.requiredWithClosedLRA
+
+    thne we can remove this method
```

Comment:
Is that issue reported upstream?

Reply:
I added https://github.com/eclipse/microprofile-lra/issues/242

Reply:
Thanks, Mike, I think that should be added into the comment in the code as you did with JBTM-3211 below

Reply:
Done